omnidocs/tasks/layout_analysis/extractors/doc_layout_yolo.py

```
@@ -70,32 +87,62 @@ def __init__(
             if self.show_log:
-                logger.info(f"Downloading model from {self.MODEL_REPO}...")
-            model_dir = snapshot_download(self.MODEL_REPO, local_dir=str(self.local_dir))
-            self.model_path = Path(model_dir) / self.MODEL_FILENAME
+                logger.info(f"Downloading YOLO model from {self.MODEL_REPO}...")
+                logger.info(f"Saving to: {self.model_path}")
+
+            # Create model directory
+            self.model_path.mkdir(parents=True, exist_ok=True)
+
+            # Download model file
+            model_dir = snapshot_download(
```

Ruff (0.12.2)

Local variable `model_dir` is assigned to but never used

Remove assignment to unused variable `model_dir`

(F841)